### ISSUE TITLE
robloxfpsunlocker only working for 5 seconds, then not working.

### ISSUE THREAD

**Issue by BotLobbyKid** [axstin/rbxfpsunlocker#1884]
When I open up a game, I can see that my game is running smoothly until 3-5 seconds into the game. My mouse stutters across the screen differencing from 3-5 seconds ago. My FPS does show that it is above 60 but my display certainly isn't. Rarely my fps unlocker runs throughout the game.

**Comment by donchan1** [comment]
is anything else choppy when you have roblox open?
this could be a polling rate issue. try changing the polling rate of your mouse to something lower.

oh and what is the refresh rate for your monitor?

**Comment by BotLobbyKid** [comment]
It's 165 hz, I tried changing the polling rate and still no difference.

**Comment by BotLobbyKid** [comment]
It's not just the mouse either, it's also the game performance.

**Comment by axstin** [comment]
Does entering fullscreen with Alt+Enter make it any better?

**Comment by BotLobbyKid** [comment]
Entering/exiting fullscreen has the same result of joining a game, making the game smooth for 3-5 seconds then stuttering for the duration of my game.

**Comment by ShadowmanWasTaken** [comment]
Try updating your graphics drivers or experimenting with the GraphicsMode options in the Roblox Studio settings.

**Comment by BotLobbyKid** [comment]
My graphics driver has been up to date for the past few weeks, still the same past result.

**Comment by LordZenkuro** [comment]
Yeah im having the same issue, it stays open untill i go to check on it in the bottom right, it doesnt work at all for some reason

**Comment by xSoundGodx** [comment]
This is my exact problem. Unless I cap to 144 or less, my game just crashes after a few seconds of being loaded in. And in some games, I have to set my FPS cap to even less than that (sometimes having to just go back to 60).
